compaction/table_state.hh

```diff
@@ -28,7 +28,7 @@ public:
     // min threshold as defined by table.
     virtual unsigned min_compaction_threshold() const noexcept = 0;
     virtual bool compaction_enforce_min_threshold() const noexcept = 0;
-    virtual const sstables::sstable_set& get_sstable_set() const = 0;
+    virtual const sstables::sstable_set& main_sstable_set() const = 0;
```

Comment:
could be changed to `const noexcept` while at it.

Reply:
Really, it adds nothing.